include/clangmetatool/include_graph_dependencies.h

```diff
@@ -45,7 +46,39 @@ struct IncludeGraphDependencies {
    */
   static std::set<clangmetatool::types::FileUID>
   liveDependencies(const clangmetatool::collectors::IncludeGraphData *data,
-                   const clangmetatool::types::FileUID &headerFUID);
+                   const clangmetatool::types::FileUID &fileFUID);
+
+  /*
+   * A data structure for include graph weak dependencies analysis
+   * for a specific source file
+   *
+   * Key: all headers the file directly depends on
```

Comment:
```suggestion
   * Key: all headers the file indirectly depends on
```

They keys would be the headers providing defnitions & value would be all the headers directly included by the source file which can lead to this header.